src/BSON/Binary.c

```diff
@@ -270,6 +296,28 @@ static PHP_METHOD(Binary, unserialize)
 	zval_ptr_dtor(&props);
 } /* }}} */
 
+/* {{{ proto array MongoDB\Driver\Binary::__serialize()
+*/
+static PHP_METHOD(Binary, __serialize)
+{
+	PHONGO_PARSE_PARAMETERS_NONE();
+
+	RETURN_ARR(php_phongo_binary_get_properties_hash(PHONGO_COMPAT_OBJ_P(getThis()), true));
+} /* }}} */
+
+/* {{{ proto array MongoDB\Driver\Binary::__unserialize()
```

Comment:
These protos are incorrect. The methods should have a `void` return annotation and take an `array $data` parameter and . I'll revise in a subsequent commit.